CHANGES_NEXT_RELEASE

```diff
@@ -20,6 +20,7 @@
 - Fix: missing error payload for TooManyResults error in 'DELETE /v2/entities/<eId>' operation (Issue #1346)
 - Fix: correctly rending as JSON error responses that were being rendered as plain text (#1989, #1991) 
 - Fix: NGSIv2 "InvalidModification" errors changed to "Unprocessable" (according to rules defined in #1286)
+- Fix: Using default values for types of Entities, Attributes, and Metadata aligned with the type names used by schema.org (Issue #2223)
```

Comment:
Add "... except for StructuredValue".

Hopefully, that will be solved before 1.3.0 get released (thus CNR entry adjusted) but in the case we fail at least the CNR entry will remain with a precise description.